params.yaml

```diff
@@ -151,7 +151,6 @@ model:
     all:
       - "meta_township_code"
       - "meta_nbhd_code"
-      - "meta_modeling_group"
```

Comment:
This feature is redundant as long as class is a feature in the model, hence the removal.